Source/Editor/Content/Items/ContentItem.cs

```diff
@@ -747,7 +742,7 @@ public override void Draw()
             
             // Draw short name
             Render2D.PushClip(ref textRect);
-            Render2D.DrawText(style.FontMedium, ShowFileExtension || view.ShowFileExtensions ? FileName : ShortName, textRect, style.Foreground, nameAlignment, TextAlignment.Center, TextWrapping.WrapWords, 1f, 0.95f);
+            Render2D.DrawText(style.FontMedium, ShowFileExtension || view.ShowFileExtensions ? FileName : ShortName, textRect, style.Foreground, nameAlignment, TextAlignment.Center, TextWrapping.WrapWords, 0.75f, 0.95f);
```

Comment:
Any reason for this change? There was a complaint a while about about spacing between the lines was not enough and some letters were too close to the next line.

Reply:
idk i will leave it at 1